sdk/tests/conformance2/context/methods-2.html

```diff
@@ -188,6 +188,7 @@
 "invalidateSubFramebuffer",
 "readBuffer",
 "renderbufferStorageMultisample",
+"texImage3D",
 "texStorage2D",
 "texStorage3D",
 "texSubImage3D",
```

Comment:
Good point -- there are multiple duplicates in the "WebGL2 methods" part of this list. Olli, do you want to run the list through "uniq" or should I do it in a subsequent pull request?